### ISSUE TITLE
[Feature] Prefixes 3/n: linear combinations of prefixes

### ISSUE THREAD

**Issue by olets** [olets/zsh-abbr#136]
Issue for documentation/findability.

- For discussion, see #114

See also

- #134
- #135

This feature is current a candidate for v6.

- Scalar and glob prefixes can be strung together

## After

```
# .zshrc
the_scalar_prefixes_array_name_tbd+=( 'xyz ' )
the_glob_prefixes_array_name_tbd=( '?.' )
```

```
% abbr a=bcd
% e.xyz sudo a[SPACE] # expands to `e.xyz sudo bcd`
% xyz e.sudo a[SPACE] # expands to `xyz e.sudo bcd`
% xyz sudo e.a[SPACE] # expands to `xyz sudo e.bcd`
```

**Comment by olets** [comment]
This is in v6.0.0-beta.1

**Comment by olets** [comment]
Released in v6.0.0
